from exceptions import Exception
class SaveInfraccionException(Exception):
	pass